canvas/image.go

```diff
@@ -40,6 +50,8 @@ type Image struct {
 
 	Translucency float64   // Set a translucency value > 0.0 to fade the image
 	FillMode     ImageFill // Specify how the image should scale to fill or fit
+
+	ScalingFilter TextureFilter // set how the image is scaled
```

Comment:
This name may relate to the question above. It's not great to repeat if it can be avoided also, but may depend on what the opinions are on the other naming question.